agent/lib/kontena/workers/stats_worker.rb

```diff
@@ -30,6 +30,9 @@ def configure_statsd(node)
       else
         @statsd = nil
       end
+    rescue => error
+      warn "statsd configuration failed: #{error.message}"
+      warn error.backtrace.join('\n') if error.backtrace
```

Comment:
If my original comment was missed: the `'\n'` prints the backtrace on a single line, this needs to be `"\n"`